### PR TITLE
added crime index and the did u mean functionality

### PR DESCRIPTION
**Pull request by roprashar**
This change is [Reviewable](https://reviewable.io/reviews/serhii73/place2live/141)

### CODE REVIEW

**Comment by serhii73** [comment]
Can you fix the build error?
```
pip install reorder-python-imports
```
and then 
```
reorder-python-imports where.py
```

**Comment by serhii73** [comment]
Thank you very much for your PR.

**Comment by roprashar** [comment]
Hei, why import reorder check fails , any idea ? Thanks

**Comment by serhii73** [comment]
It's a reason because I added `reorder-python-imports` to `pre-commit` config.
And if `reorder-python-imports` will update a file will be shown a build error.

**Comment by serhii73** [comment]
Maybe exist better solution if I want all python files to follow `PEP8 import` in this repo.

**Comment by roprashar** [comment]
I corrected , it should good to go now

**Comment by serhii73** [comment]
Thank you very much!
I will check it soon.

File: where.py [hunk 1]
```diff
@@ -121,7 +131,8 @@ def safety_func():
     )
     print(
         text_color(
-            f"In your country safety index is {country_safety_index}",
+            f"In your country safety index is {country_safety_index} or "
+            f" crime index is {round((100.0-country_safety_index), 2)}",
```
Comment (serhii73) [inline where.py]: Is it a formula? How to count `safe index`?
100 - `safety index`?

File: where.py [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
 
 from utils import text_color
 from utils import text_type
 
-
 df = pd.read_csv("city/output/list_of_countries.csv")
 
 
+def get_closest_country(country_name: str):
+    """ This function finds the closest match for the country"""
```
Comment (serhii73) [inline where.py]: Why the space after `"""`?

File: where.py (hunk 1 above)
Comment (roprashar) [reply where.py]: yes this is exactly, how we get crime index form safety index

File: where.py (hunk 2 above)
Comment (serhii73) [reply where.py]: Also, please, write a point at the end of the sentence.

File: where.py [hunk 3]
```diff
@@ -20,9 +28,11 @@ def run_country_checker():
             country = country.title()
             float(df[df.country == country]["purchasing_power_index"])
         except TypeError:
+            res = get_closest_country(country)
             print(
                     text_color(
-                        f"'{country}' is an invalid country. Please try again.",
+                        f"'{country}' is an invalid country or did you mean {res}"
```
Comment (serhii73) [inline where.py]:
```
'Adasdasd' is an invalid country or did you mean []. Please try again.
```
Maybe improve the output message? What do you think about that?

File: where.py (hunk 2 above)
Comment (roprashar) [reply where.py]: i fixed !

File: where.py [hunk 4]
```diff
@@ -1,12 +1,20 @@
...
 from utils import text_type
 
-
 df = pd.read_csv("city/output/list_of_countries.csv")
 
 
+def get_closest_country(country_name: str):
+    """ This function finds the closest match for the country"""
+    countries = df['country'].values.tolist()
+    results = difflib.get_close_matches(country_name, countries)
```
Comment (serhii73) [inline where.py]: Maybe to rename `results`?
For example - `close_matches_countries` or `close_matches_countries_list`?
Or maybe exist a better option?

File: where.py (hunk 1 above)
Comment (serhii73) [reply where.py]: I think it's not necessary. Why need showing `crime index`? If exist `safety index`  🤔

**Comment by serhii73** [comment]
Can you please resolve the Git conflict?
Thank you very much!

**Comment by roprashar** [comment]
please let me know if any changes required .

**Pull request merged**